step-controller/step-controller-server/src/main/java/step/core/Controller.java

```diff
@@ -131,8 +131,10 @@ private void initContext() throws ClassNotFoundException, PluginManager.Builder.
 				collectionFactory.getCollection("executions", Execution.class));
 		context.setExecutionAccessor(executionAccessor);		
 		context.setExecutionManager(new ExecutionManagerImpl(executionAccessor));
-		
-		context.setPlanAccessor(new PlanAccessorImpl(collectionFactory.getCollection("plans", Plan.class)));
+
+		PlanAccessorImpl plans = new PlanAccessorImpl(collectionFactory.getCollection("plans", Plan.class));
```

Comment:
No change?

Reply:
the versioning is enabled in an EE plugin